actionview/lib/action_view/helpers/tag_helper.rb

```diff
@@ -70,6 +70,7 @@ def tag_string(name, content = nil, escape: true, **options, &block)
           content = @view_context.capture(self, &block) if block_given?
           self_closing = SVG_SELF_CLOSING_ELEMENTS.include?(name)
           if (HTML_VOID_ELEMENTS.include?(name) || self_closing) && content.nil?
+            TagHelper.ensure_valid_html5_tag_name(name.to_s)
```

Comment:
We don't need this line, since the tag name must be present in HTML_VOID_ELEMENTS or SVG_SELF_CLOSING_ELEMENTS and is therefore valid.